File: Atcoder_#337/javascript/a_scoreboard.js
function main(input) {
    const lines = input.trim().split('\n');
    const N = parseInt(lines[0], 10);
    let totalTakahashi = 0;
    let totalAoki = 0;

    for (let i = 1; i <= N; i++) {
        const [X, Y] = lines[i].split(' ').map(Number);
        totalTakahashi += X;
        totalAoki += Y;
    }

    if (totalTakahashi > totalAoki) {
        console.log('Takahashi');
    } else if (totalTakahashi < totalAoki) {
        console.log('Aoki');
    } else {
        console.log('Draw');
    }
}

const fs = require('fs');
const input = fs.readFileSync('/dev/stdin', 'utf8');
main(input);

File: Atcoder_#337/ruby/a_scoreboard.rb
n = gets.to_i
scores_takahashi = []
scores_aoki = []

n.times do
  x, y = gets.split.map(&:to_i)
  scores_takahashi << x
  scores_aoki << y
end

# 合計点を計算
total_takahashi = scores_takahashi.sum
total_aoki = scores_aoki.sum

# 結果を出力
if total_takahashi > total_aoki
  puts 'Takahashi'
elsif total_takahashi < total_aoki
  puts 'Aoki'
else
  puts 'Draw'
end
